### PR TITLE
setup project configuration

### PR DESCRIPTION
**Pull request by Antonieta13**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @renardete @holgiosalos @Valeryibarra
```
Comment (Valeryibarra) [inline .github/CODEOWNERS]: EOF

**Review by holgiosalos — CHANGES_REQUESTED** [review]
Please also add the .gitignore file to the repo (or create if it does not exist) Step 3 of first section
https://github.com/renardete/Cypress-Training-Doc#1-creaci%C3%B3n-y-configuraci%C3%B3n-del-repositorio

File: .github/CODEOWNERS (hunk 1 above)
Comment (Antonieta13) [reply .github/CODEOWNERS]: Listo @Valeryibarra

File: .gitinore [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+*.msm
+*.msp
+
+# Windows shortcuts
+*.lnk
+
+# End of https://www.toptal.com/developers/gitignore/api/vscode,node,windows,cypressio
+
+
+cypress/integration/examples/*
```
Comment (holgiosalos) [inline .gitinore]: Please move the `cypress/integration/examples/*` to the section where the Cypress patterns are, that is between lines 7 and 12

**Review by holgiosalos — CHANGES_REQUESTED** [review]

File: .gitinore (hunk 2 above)
Comment (Antonieta13) [reply .gitinore]: Listo

**Review by holgiosalos — APPROVED** [review]

**Pull request merged**